packs/feats/receive-prayers.json

```diff
@@ -28,11 +28,23 @@
             "remaster": true,
             "title": "Pathfinder War of Immortals"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "receive-prayers",
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "action:sense-motive"
```

Comment:
Did you intend to include `"receive-prayers"` in the predicate?

Reply:
I did, my bad.